### ISSUE TITLE
Large Start Button & Taskbar Icons in Windows 10

### ISSUE THREAD

**Issue by SL6464** [Open-Shell/Open-Shell-Menu#530]
Title: Large Start Button & Taskbar Icons in Windows 10

**Is your feature request related to a problem? Please describe.**
The Windows 10 start button and taskbar icons are smaller than they were in Windows 8.1.  

There are options in other programs such as 7+ Taskbar Tweaker to increase the size of the 

taskbar icons, but not the start button.  StartIsBack allows both to be increased to the 

larger size.

**Describe the solution you'd like**
The option to increase the size of the start button and taskbar items to match what they 

were in Windows 8.1.

**Describe alternatives you've considered**
The only alternative is to use StartIsBack, or 7+ Taskbar Tweaker and a replacement start 

button in Open-Shell (which has its own problems and isn't a solution I'm fond of).  I find 

Open-Shell to be superior to StartIsBack in most ways, but this one thing has kept me using 

StartIsBack on my Windows 10 machine.

**Additional context**
Windows 8.1: https://i.imgur.com/Qi2BrbV.png

Windows 10 (with either no replacement start menu or Open-Shell): 

https://i.imgur.com/HMrjriS.png

Windows 10 with StartIsBack: https://i.imgur.com/sx30AVO.png

You'll note that the larger 10 start button is still a bit smaller than the 8.1 start 

button.  Not sure why that is.

Please consider adding this feature as it would make a huge difference for me in terms of 

how much I like using Open-Shell, and going by StartIsBack and 7+ Taskbar Tweaker, it's 

definitely something that can be done.

**Comment by NebelNidas** [comment]
Related: #249

**Comment by TheDolphinSlayer** [comment]
Is it possible to also include taskbar skinning? (By taskbar skinning, I mean the taskbar buttons, tray icons, etc) and the possibility of bringing back "classic" features of the taskbar? For example, the clock from win7/win8? not entirely sure how those at startisback++ does it, but when the registry is added to restore the clock in windows 10, it only works with startisback++.
I really think open shell could definitely pull this off, making this program the ultimate skinner for any Windows OS. Honestly, the only reason I don't use OpenShell is that the program conflicts with StartisBack++, or WindowsBlinds. Or even just a proper switch to completely turn off the taskbar customization option as (even when the one that is there now is turned off) still breaks the taskbar stopping startisback++ and WindowsBlinds from functioning properly on the taskbar. Now, Classic Shell, 4.3.0 is the last release that allows taskbar skinning from other programs, as well as the use of the classic shell start menu. Though newer skins being made is incompatible with that old one, so i'm conflicted. But having fully functional themed taskbar, aswell as some options for restoring classic features, would completely sell open shell to me.

**Comment by ge0rdi** [comment]
@TheDolphinSlayer 
> Or even just a proper switch to completely turn off the taskbar customization option as (even when the one that is there now is turned off) still breaks the taskbar stopping startisback++ and WindowsBlinds from functioning properly on the taskbar.

Could you try latest nightly build?

It should not to any taskbar-related hooking when taskbar customization is disabled.
It fixed WindowsBlinds compatibility (according to reports so far), I guess it should fix also StartIsBack++ compatibility.

**Comment by TheDolphinSlayer** [comment]
Oh, it works great! The only question now is how do I access the settings page? as StartisBack++ takes over the context menus
________________________________

**Comment by ge0rdi** [comment]
Thanks for checking out.

You can access settings using `Open-Shell Menu Settings` shortcut in Start menu.

**Comment by bonzibudd** [comment]
To clarify: Are you able to use all of the taskbar customization from StartIsBack now?

Also, you may be able to use the OSM context menu if you set it to open on Shift-Right click. I haven’t tested this myself though.

**Comment by TheDolphinSlayer** [comment]
Yeah, StartisBack++ works great! Honestly would be easier if open she'll just had all of these features (though I know thats 100% easier said than done)

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by bonzibudd** [comment]
Agreed, but at least it’s now possible for the programs to work together.

One thing I am curious about is the classic Taskbar clock “Flyout.” From what I remember, the code was actually removed from Win10, but somehow StartIsBack manages to bring it back from the dead. Possibly the files/dlls are still there, it just requires some little-known injection process to activate it. But I wouldn’t be surprised if the creator of StartIsBack just recreated the entire thing, just like he did for other features, like the jumplists and thumbnails.

I am personally still happy with the workarounds for when I need them, so I guess I’ll leave it here :)
